data/virt_autotest/virt-tests.conf

```diff
@@ -0,0 +1,136 @@
+[vmi_lifecycle_test]
```

Comment:
Could you help move this file into a 'kubevirt folder' in data/virt_autotest if this belongs to kubevirt or something else ? I think basic category is necessary for clarity.

Reply:
Do you mean to create a dedicate folder under data/virt_autotest?

Reply:
Yes, for your kubvirt test.

Reply:
There is only 1 config file for kubevirt test, so I think it's not necessary to create a dedicated folder for now.

Reply:
As I said, the file name itself 'virt-tests.conf' does not tell what this file is used for. So if you do not move it to a subfolder, please rename it to something more meaningful otherwise you need to create a subfolder for it.

Reply:
OK, already renamed as 'kubevirt-tests.conf'.